panels/config/config-entries/ha-config-entries.html

```diff
@@ -13,60 +13,84 @@
   <template>
     <style include='iron-flex ha-style'>
       .content {
-        padding: 8px;
+        padding-bottom: 24px;
       }
-      .entries {
-        margin: 0 -4px;
-      }
-      .entries paper-card {
-        margin: 4px;
+      paper-button {
+        color: var(--primary-color);
+        font-weight: 500;
+        top: 3px;
+        height: 37px;
+        margin-right: -.57em;
       }
     </style>
+
     <hass-subpage title='Integrations'>
-      <div class='content'>
-        <h1>Configured</h1>
-        <template is='dom-if' if='[[!_entries.length]]'>
-          <i>Nothing configured.</i>
-        </template>
-        <div class='entries layout horizontal wrap'>
-          <template is='dom-repeat' items='[[_entries]]'>
-            <paper-card heading='[[item.title]]'>
+      <template is='dom-if' if='[[_progress.length]]'>
+        <div class='content'>
+          <ha-config-section>
```

Comment:
We should pass in `is-wide`, that way the config section knows when to shrink on small devices.